### ISSUE TITLE
Lack of focus ring on Payment variant selector on the landing page

### ISSUE THREAD

**Issue by BANOnotIT** [jolbol1/jolly-ui#84]
![Image](https://github.com/user-attachments/assets/fcd6a4a9-9dd7-4f36-a01f-039369378bbc)

Payment selector only has "selected" style, but no "focused" style. So there's no way for me as a keyboard user to know where my focus is until I tap `Tab` several times and find the focus ring on some other element.

[SC 2.4.7](https://www.w3.org/TR/WCAG21/#focus-visible)

**Comment by BANOnotIT** [comment]
Sadly, the problem is only solved for the first item in the group, second and third still have no focused style.

https://github.com/user-attachments/assets/2c5f8297-ca20-4c69-bb1b-23f198846861

**Comment by jolbol1** [comment]
Yep, my bad, even broke the hover style too. Should be fixed now thanks 👍
